recipe/patches/0001-disable-requirement-on-gym-notices.patch

```diff
@@ -0,0 +1,25 @@
+From 177f184223f25bc517ff45d81caee1a9b7f2175a Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Thu, 15 Dec 2022 11:14:17 +1100
+Subject: [PATCH] disable requirement on gym-notices
+
+---
+ setup.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index fc4d9f94..16bf2064 100644
+--- a/setup.py
++++ b/setup.py
+@@ -65,7 +65,7 @@ setup(
+         "numpy >= 1.18.0",
+         "cloudpickle >= 1.2.0",
+         "importlib_metadata >= 4.8.0; python_version < '3.10'",
+-        "gym_notices >= 0.0.4",
++        # "gym_notices >= 0.0.4",
```

Comment:
Update to gymnasium-notices

Reply:
Please see [this](https://github.com/conda-forge/gymnasium-feedstock/pull/9#issuecomment-1352518641) comment.